Index: src/test/java/Pages/UserDashboard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Pages;\r\n\r\npublic class UserDashboard {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/Pages/UserDashboard.java b/src/test/java/Pages/UserDashboard.java
--- a/src/test/java/Pages/UserDashboard.java	(revision b802f7fb884bc115ba2e61e1eaf5cec4791ced2b)
+++ b/src/test/java/Pages/UserDashboard.java	(date 1725549786909)
@@ -1,4 +1,11 @@
 package Pages;
 
+import org.openqa.selenium.support.PageFactory;
+
 public class UserDashboard {
+
+    public UserDashboard(){
+        PageFactory.initElements(utilities.Driver.getDriver(), this);
+    }
+
 }
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>target/\r\ntest-output/\r\n!.mvn/wrapper/maven-wrapper.jar\r\n!**/src/main/**/target/\r\n!**/src/test/**/target/\r\n\r\n### IntelliJ IDEA ###\r\n.idea/modules.xml\r\n.idea/jarRepositories.xml\r\n.idea/compiler.xml\r\n.idea/libraries/\r\n*.iws\r\n*.iml\r\n*.ipr\r\n\r\n### Eclipse ###\r\n.apt_generated\r\n.classpath\r\n.factorypath\r\n.project\r\n.settings\r\n.springBeans\r\n.sts4-cache\r\n\r\n### NetBeans ###\r\n/nbproject/private/\r\n/nbbuild/\r\n/dist/\r\n/nbdist/\r\n/.nb-gradle/\r\nbuild/\r\n!**/src/main/**/build/\r\n!**/src/test/**/build/\r\n\r\n### VS Code ###\r\n.vscode/\r\n\r\n### Mac OS ###\r\n.DS_Store
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision b802f7fb884bc115ba2e61e1eaf5cec4791ced2b)
+++ b/.gitignore	(date 1725549786925)
@@ -12,6 +12,7 @@
 *.iws
 *.iml
 *.ipr
+configuration.properties
 
 ### Eclipse ###
 .apt_generated
Index: configuration.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\r\n# configuration.properties dosyasini kisiye özel kullanacagiz. Bu yüzden;\r\n# Bu kisma kendi email/pass bilgilerinizi girip.\r\n# Sonrasinda bilgilerin diger kisilerle karismamasi için\r\n# configuration.properties sayfasini gitignore dosyasina ekleyin.\r\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\r\n\r\n\r\n\r\nbrowser=chrome\r\n\r\nWebsiteMainPageURL=http://qa.hauseheaven.com\r\nAdminDashUrl = https://qa.hauseheaven.com/admin\r\n\r\n\r\nWebsiteGecerliEmail = onurvardal@hauseheaven.com\r\nWebsiteGecerliPassword = Hven.150820\r\n\r\nAdminGecerliEmail = onurvardal.admin@hauseheaven.com\r\nAdminGecerliPassword = Hven.150820\r\n\r\nItemName: autotest_onur\r\nItemNameEdit: autotest_onur1\r\nDescriptionEdit: autotest_edit\r\nStatusEdit: Draft\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/configuration.properties b/configuration.properties
--- a/configuration.properties	(revision b802f7fb884bc115ba2e61e1eaf5cec4791ced2b)
+++ b/configuration.properties	(date 1725549786925)
@@ -13,10 +13,10 @@
 AdminDashUrl = https://qa.hauseheaven.com/admin
 
 
-WebsiteGecerliEmail = onurvardal@hauseheaven.com
+WebsiteGecerliEmail = canyilmaz@hauseheaven.com
 WebsiteGecerliPassword = Hven.150820
 
-AdminGecerliEmail = onurvardal.admin@hauseheaven.com
+AdminGecerliEmail =
 AdminGecerliPassword = Hven.150820
 
 ItemName: autotest_onur
Index: src/test/java/Tests/Can/US_06.java
===================================================================
diff --git a/src/test/java/Tests/Can/US_06.java b/src/test/java/Tests/Can/US_06.java
deleted file mode 100644
--- a/src/test/java/Tests/Can/US_06.java	(revision b802f7fb884bc115ba2e61e1eaf5cec4791ced2b)
+++ /dev/null	(revision b802f7fb884bc115ba2e61e1eaf5cec4791ced2b)
@@ -1,4 +0,0 @@
-package Tests.Can;
-
-public class Can_US_06_07_08_19 {
-}
